defaults/main.yml

```diff
@@ -27,7 +27,8 @@ roundcube__nginx_access_policy: ''
 # List of PHP5 packages required by Roundcube.
 roundcube__php5_packages: [ 'php-auth-sasl', 'php5-gd', 'php5-intl', 'php5-json',
                            'php5-mcrypt', 'php-mail-mime', 'php-mail-mimedecode',
-                           'php-net-smtp', 'php-net-socket', 'php-pear' ]
+                           'php-net-smtp', 'php-net-socket', 'php-pear',
+                           'php-net-idna2', 'php-net-ldap3' ]
```

Comment:
- `php-net-idna2` is not part of Debian Jessie (which is the primary platform targeted by DebOps) or Debian Jessie-Backports therefore I cannot add this to the default package set. This is definitely a bug in the role as the library is listed in the upstream installation requirements. I guess the proper fix would be to install it with composer on platforms which don't have it packaged. Feel free to add it to `roundcube__extra_packages` in your Ansible inventory if you can install it via regular package manager.
- `php-net-ldap3` is only used with a LDAP address book. I usually configure the address book via carddav and therefore don't use this package. Also this one can therefore be added in the `roundcube__extra_packages` by the people who use it.

I feel that DebOps roles shouldn't install more software than is absolutely need for the software to work. If people want to customize the installation for their own use case, this should be supported by the role without changing the code. I think that's already the case here.